charts/nextcloud/templates/ingress.yaml

```
@@ -9,37 +9,37 @@ metadata:
     app.kubernetes.io/instance: {{ .Release.Name }}
     app.kubernetes.io/managed-by: {{ .Release.Service }}
     app.kubernetes.io/component: app
-{{- if .Values.ingress.labels }}
+    {{- if .Values.ingress.labels }}
 {{ toYaml .Values.ingress.labels | indent 4 }}
```

```suggestion
    {{- toYaml .Values.ingress.labels | indent 4 }}
```
Wouldn't it make sense to also indent this for better readability?

---

Or am I misunderstanding you and you want to do this in a follow-up PR?

---

in the following PR (and there replace `if` with `with`)